hubspot3/crm_associations.py

```diff
@@ -179,3 +182,62 @@ def link_owner_to_company(self, owner_id, company_id):
             to_object=company_id,
             definition=Definitions.OWNER_TO_COMPANY,
         )
+
+    def chunks(self, it, size=100):
+        l = len(it)
+        it = iter(it)
+        for i in range(0, int(l / size) + (l % size > 0)):
```

Comment:
`(l % size > 0)` o restultado disso não vai ser um booleano? Como está sendo considerado na soma?

Reply:
Vai sim, se for True ele soma 1, se não soma 0. Fiz uns testes pra ver se realmente funcionava porque a principio também não acreditei. kkk